src/hooks/useAuthenticated.ts

```diff
@@ -1,11 +1,11 @@
 import { useEffect, useState } from 'react';
-import { USER_SELF, DEFAULT_AVATAR, SIGNUP_LINK } from '@/constants/url';
-import fetch from '@/helperFunctions/fetch';
+import { DEFAULT_AVATAR, SIGNUP_LINK } from '@/constants/url';
+import useUserData from './useUserData';
 
 type Userdata = {
-    userName: string;
-    firstName: string;
-    profilePicture: string;
+    userName: string | undefined;
+    firstName: string | undefined;
+    profilePicture: string | undefined;
 };
```

Comment:
Please write these types into interface folder and from there export here and use it and do it every where in your PR

Reply:
Done!